ffi-build.sh

```diff
@@ -64,7 +64,13 @@ actual_native_static_libs="$(cargo rustc --release --target ${target} -- --print
 echo "Actual native-static-libs:${actual_native_static_libs}"
 echo "Expected native-static-libs:${expected_native_static_libs}"
 
-[ "${expected_native_static_libs}" = "${actual_native_static_libs}" ]
+# compute libs in $actual_native_static_libs but not in $expected_native_static_libs
+unexpected_native_libs=$(comm -13 <(echo $expected_native_static_libs | sed 's/ -/\n-/g' | sort -u) <(echo $actual_native_static_libs | sed 's/ -/\n-/g' | sort -u))
```

Comment:
[minor] With the new implementation, it is no longer relevant to repeat the libraries. We can just add a simple of unique elements ?

Reply:
Indeed, I can remove the duplicates from expected_native_libs.